common/static/coffee/spec/discussion/view/discussion_view_spec_helper.coffee

```diff
@@ -23,6 +23,15 @@ class @DiscussionViewSpecHelper
         }
         $.extend(thread, props)
 
+    @checkVoteClasses = (view) ->
+        debugger;
+        view.render()
+        display_button = view.$el.find(".display-vote")
+        expect(display_button.hasClass("is-hidden")).toEqual(true)
```

Comment:
I think you want `toBe(true)`. Otherwise you can use `toBeTruthy`: http://jasmine.github.io/1.3/introduction.html#section-Included_Matchers